wiki-countries-app/src/App.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import Navbar from './components/Navbar'
+import CountriesList from './components/CountriesList'
+import CountriesDetails from './components/CountryDetails'
+import 'bootstrap/dist/css/bootstrap.css';
+import './App.css';
+
+import { Switch, Route, Redirect } from 'react-router-dom'
+
+function App() {
+  return (
+    <>
+      <Navbar />
+      <div className="container">
+        <div className="row">
+          <div className="col-5">
+            <div className="list-group">
+              <Route path="/" render={() => <CountriesList />} />
+            </div>
+          </div>
+          <div className="col-7">
+            <Route path="/:cca3" render={(props) => <CountriesDetails {...props} />} />
```

Comment:
```suggestion
<Switch>
            <Route exact path="/" render={() => <CountriesList />} />
            <Route
              exact
              path="/:cca3"
              render={(props) => <CountriesDetails {...props} />}
            />
          </Switch>
```
En este caso solo teníais una ruta, pero debéis hacer uso del _Switch_ de _react-router-dom_ para envolver todas las rutas, ya que será este quien gestione a cual "entrar" en cada momento según el path de la URL.